framework/src/modules/chain/components/storage/sql/block_temp/create.sql

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+INSERT INTO block_temp (
+	${fields:raw}
```

Comment:
As we have always fixed set of fields, then would prefer to hardcode the field names in SQL. When you have dynamic fields based on conditions then we calculate it JS and pass it as params.

Reply:
Can you give me an example? I looked at create queries for other entities and they follow the same practice. I am not sure how you'd like me to change this particular query.